packages/tools/plymouth/package.mk

```diff
@@ -0,0 +1,100 @@
+################################################################################
+#      This file is part of OpenELEC - http://www.openelec.tv
+#      Copyright (C) 2009-2016 Stephan Raue (stephan@openelec.tv)
+#
+#  OpenELEC is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 2 of the License, or
+#  (at your option) any later version.
+#
+#  OpenELEC is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with OpenELEC.  If not, see <http://www.gnu.org/licenses/>.
+################################################################################
+
+PKG_NAME="plymouth"
+PKG_VERSION="b9f2aac5"
+PKG_REV="1"
+PKG_ARCH="any"
+PKG_LICENSE="GPL"
+PKG_SITE="https://cgit.freedesktop.org/plymouth/"
+PKG_URL="https://gitlab.freedesktop.org/plymouth/plymouth/-/archive/master/plymouth-$PKG_VERSION.tar.gz"
```

Comment:
```suggestion
PKG_URL="https://gitlab.freedesktop.org/plymouth/plymouth/-/archive/master/plymouth-$PKG_VERSION.tar.bz2"
```
bz2 is smaller then gz :)  (idk why they use legacy codecs but at least better then gz)